circle.yml

```diff
@@ -1,17 +1,16 @@
 machine:
-  pre:
-    - ./scripts/circle-ci/install-docker.sh:
-        pwd: $CIRCLE_PROJECT_REPONAME
-    - sudo wget --retry-connrefused --waitretry=1 --read-timeout 20 --timeout 15 -t 10 -q -O /usr/local/bin/docker-compose https://github.com/docker/compose/releases/download/1.9.0/docker-compose-`uname -s`-`uname -m`
-    - sudo chmod +x /usr/local/bin/docker-compose
   java:
     version:
       oraclejdk8
   environment:
     _JAVA_OPTIONS: "-Xmx512M -XX:+HeapDumpOnOutOfMemoryError -verbose:gc -XX:+PrintGC -XX:+PrintGCDateStamps -XX:+PrintGCDetails -XX:-TraceClassUnloading -Xloggc:build-%t-%p.gc.log"
     TERM: dumb
-  services:
-    - docker
+  post:
```

Comment:
Added (environment variables - whether we're running on internal circle or not - aren't set in the pre stage)